R/pkg/R/functions.R

```diff
@@ -2095,26 +2061,28 @@ setMethod("atan2", signature(y = "Column"),
             column(jc)
           })
 
-#' datediff
+#' @section Details:
+#' \code{datediff}: Returns the number of days from \code{start} to \code{end}.
 #'
-#' Returns the number of days from \code{start} to \code{end}.
-#'
-#' @param x start Column to use.
-#' @param y end Column to use.
```

Comment:
two concerns here:
- the doc description is now fairly generalized - is supposed to be `datediff(end, start)` but seems like it's not not clear which is end and which is start
- renaming this could break user calling by name `datediff(df$c, x = "foo")`